vector/src/main/java/im/vector/riotx/features/notifications/NotificationDrawerManager.kt

```diff
@@ -343,12 +343,19 @@ class NotificationDrawerManager @Inject constructor(private val context: Context
                         globalLastMessageTimestamp = lastMessageTimestamp
                     }
 
+                    val tickerText = if (roomEventGroupInfo.isDirect) {
+                        String.format("%s: %s", events[0].senderName, events[0].description)
```

Comment:
Why using the first event? I think this will be the oldest one. We should maybe use the latest one. I let you decide what is best

Reply:
Let me think about that for a bit. You're probably right.